options/posix/include/sys/param.h

```diff
@@ -3,6 +3,11 @@
 #define _SYS_PARAM_H
 
 #include <endian.h>
+#include <limits.h>
+
+#if !defined NGROUPS && defined NGROUPS_MAX
+# define NGROUPS NGROUPS_MAX
+#endif
```

Comment:
This is quite ugly. Whether the value is defined or not depends on the order of the headers?

Reply:
Yeah I don’t like it either, but this is what Linux does. I think we can unconditionally define them

Reply:
I guess that would be preferable.